net-im/telegram-desktop-bin/telegram-desktop-bin-6.2.4.ebuild

```diff
@@ -17,6 +17,7 @@ S="${WORKDIR}/Telegram"
 LICENSE="GPL-3-with-openssl-exception"
 SLOT="0"
 KEYWORDS="-* ~amd64"
+IUSE="webkit"
```

Comment:
This likely needs a rename of the ebuild to -r1

Reply:
Thanks for the quick review! I think revision bump is not necessary... because adding the new USE will trigger a rebuild.